cmd/swctl/app/cmd_dependency.go

```
@@ -255,10 +312,15 @@ func (*Dependencies) ResizeHugePages(cli Cli, size uint) error {
 	if err != nil {
 		return err
 	}
+	allocatedHP, _ := AllocatedHugePages(cli)
```

Don't ignore error from function.